app/javascript/packs/provider_active_docs.ts

```diff
@@ -0,0 +1,18 @@
+// We can define the 3scale plugins here and export the modified bundle
+import SwaggerUI from 'swagger-ui'
+import 'swagger-ui/dist/swagger-ui.css'
+
+import { autocompleteOAS3 } from 'ActiveDocs/OAS3Autocomplete'
+
+import 'ActiveDocs/swagger-ui-3-patch.scss'
+
+const accountDataUrl = '/p/admin/api_docs/account_data.json'
```

Comment:
This file seems to be the exact copy of `app/javascript/packs/active_docs.ts`, only the `accountDataUrl` value changes.

I have no idea how these packs work, but I am wondering if there is way to parameterize this somehow in order to avoid duplicate code.

Maybe @josemigallas can help.